packages/api/src/external/carequality/patient.ts

```diff
@@ -62,43 +47,12 @@ export async function discover(patient: Patient, facilityNPI: string): Promise<v
     log(`Kicking off patient discovery`);
     await iheGateway.startPatientDiscovery(pdRequest);
 
-    const raceControl: RaceControl = { isRaceInProgress: true };
-    // Run the patient discovery until it either times out, or all the results are in the database
-    const raceResult = await Promise.race([
-      controlDuration(
-        PATIENT_DISCOVERY_TIMEOUT.asMilliseconds(),
-        `Patient discovery reached timeout after ${PATIENT_DISCOVERY_TIMEOUT.asMilliseconds()} ms`
-      ),
-      checkIfRaceIsComplete(
-        () => isPDComplete(pdRequest.id, numGateways),
-        raceControl,
-        `Patient discovery results came back in full (${pdRequest.gateways.length} gateways)`,
-        CHECK_DB_INTERVAL.asMilliseconds()
-      ),
-    ]);
-    const pdResults = await getOutboundPatientDiscoveryResps(pdRequest.id, "success");
```

Comment:
With this not being called anymore, it seems we can remove that function/file

Reply:
removed file